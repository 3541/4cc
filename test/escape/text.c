int main(void) {
    puts("Hello,\nworld\non\nmultiple\nlines.");
    return 0;
}
