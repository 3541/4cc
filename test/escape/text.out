Hello,
world
on
multiple
lines.
